src/css/override_components.scss

```diff
@@ -135,15 +135,20 @@ button,
   }
 } /* stylelint-enable */
 
+.usa-button-disabled {
```

Comment:
Hmmm, is it required for that button? or if using the `disabled` attribute on the button would be enough?